test/extension.test.ts

```diff
@@ -81,6 +93,10 @@ suite('Kits test', async () => {
     expect(nil).to.be.null;
   });
 
+  test('Scan non exisiting dir for kits', async() => {
+    const kits = await kit.scanDirForCompilerKits("");
```

Comment:
I think we should use `''` for all strings.

Reply:
Done